src/helm/benchmark/run_specs.py

```diff
@@ -2243,6 +2378,116 @@ def get_anthropic_hh_rlhf_spec(num_respondents: int, subset: str) -> RunSpec:
     )
 
 
+@run_spec_function("cleva")
+def get_cleva_spec(task: str, version: str, subtask: str = None, prompt_id: int = 0) -> RunSpec:
```

Comment:
Is there documentation somewhere about the valid options for `prompt_id`?

Reply:
We list all available `prompt_id` in `run_specs_cleva_v1.conf`. This should help users find all valid `prompt_id` options.